spec/helpers/patient_phone_number_helper_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "rails_helper"
+
+RSpec.describe PatientPhoneNumberHelper, type: :helper do
```

Comment:
Can move these specs to `spec/models/concerns/phone_number_localizable` as well, and test both `PatientPhoneNumber` and `PhoneNumberAuthentication` there.

Reply:
okay